Model/Config/Backend/Enable.php

```diff
@@ -56,6 +56,15 @@ public function afterSave()
                 \Extend\Integration\Service\Extend::ENABLE_PRODUCT_PROTECTION_CART_OFFER,
                 0
             );
+            $this->writer->save(
+                \Extend\Integration\Service\Extend::ENABLE_PRODUCT_PROTECTION_PRODUCT_DISPLAY_PAGE_OFFER,
+                0
+            );
+
+            // need to clean these cache types in order to immediately propagate the new settings to the storefront
+            // and eliminate the need for the admin to flush the cache manually in order to see all the changes in the UI.
+            $this->cacheTypeList->cleanType('config');
```

Comment:
This is already done in parent::afterSave(), and it's parent as well.

Reply:
ok. seems like it's only the `full_page` cache that needs to be cleaned here then